corehq/apps/data_analytics/views.py

```diff
@@ -0,0 +1,9 @@
+from corehq.apps.data_analytics.models import MALTRow
+from corehq.apps.data_analytics.admin import MALTRowAdmin
+from dimagi.utils.django.management import export_as_csv_action
+
+
+def malt_csv_response(month, year):
```

Comment:
I think you should move it because as-is it's confusing. putting a function inside `views.py` that doesn't take in a request and is not intended to actually be used as a view is just going to trip people up when they try and reason with the code.